tests/neg/i1688.scala

```diff
@@ -0,0 +1,6 @@
+package scala
+sealed trait Null  // error
+
+object Null  // error
+
+class AnyVal // error
```

Comment:
This gets through compilation and prints `5`:

```scala
object scala {
  trait Boolean {
    def x: Int
  }
}

object Test {
  def x: scala.Boolean = new scala.Boolean { val x = 5 }

  def main(args: Array[String]): Unit = println(x.x)
}
```

Not sure if the first object lands in `_root_` or "<empty>" ....